src/c_kzg_4844.c

```diff
@@ -1417,24 +1435,30 @@ static C_KZG_RET verify_kzg_proof_batch(
     if (ret != C_KZG_OK) goto out;
     ret = new_fr_array(&r_times_z, n);
     if (ret != C_KZG_OK) goto out;
+    ret = new_g1_array(&proofs_g1, n);
+    if (ret != C_KZG_OK) goto out;
 
     /* Compute the random lincomb challenges */
-    ret = compute_r_powers(
-        r_powers, commitments_g1, zs_fr, ys_fr, proofs_g1, n
-    );
+    ret = compute_r_powers(r_powers, claims, n);
     if (ret != C_KZG_OK) goto out;
 
+    /* copy the proofs into a single array, because g1_lincomb_naive requires
```

Comment:
Let's capitalize the first letter of every comment as we are doing everywhere.
```suggestion
    /* Copy the proofs into a single array, because g1_lincomb_naive requires
```